src/include/hip/rpp_hip_common.hpp

```diff
@@ -910,7 +910,6 @@ __device__ __forceinline__ void rpp_hip_pack_float8_and_store8(half *dstPtr, d_f
 __device__ __forceinline__ void rpp_hip_pack_float24_pkd3_and_store24_pkd3(uchar *dstPtr, d_float24 *dstPtr_f24)
 {
     d_uint6 dst_ui6;
-
```

Comment:
Lets retain this one line gap like other functions in this file. Same with L974,1053,1123

Reply:
Done